components/tidb_query_vec_expr/src/rpn_expr/mod.rs

```diff
@@ -19,12 +19,13 @@ pub mod impl_time;
 
```

Comment:
It seems that the whole `tidb_query_vec_expr/src/rpn_expr` directory needs to be deleted